bench/wtperf/config.c

```diff
@@ -653,6 +666,43 @@ config_sanity(CONFIG *cfg)
 }
 
 /*
+ * config_to_file --
+ *	Write the final config used in this execution to a file.
+ */
+void
+config_to_file(CONFIG *cfg)
+{
+	CONFIG_QUEUE_ENTRY *config_line;
+	FILE *fp;
+	size_t req_len;
+	char *path;
+
+	/* Backup the config */
+	req_len = strlen(cfg->home) + 100;
+	if ((path = calloc(req_len, 1)) == NULL) {
+		(void)enomem(cfg);
+		return;
+	}
+
+	snprintf(path, req_len + 14, "%s/CONFIGBACKUP", cfg->home);
```

Comment:
test/format calls the equivalent file ``CONFIG``, how about calling this one ``CONFIG.wtperf``